edx_django_utils/monitoring/tests/code_owner/test_utils.py

```diff
@@ -24,18 +21,6 @@ def setUp(self):
         super().setUp()
         clear_cached_mappings()
 
-    @override_settings(CODE_OWNER_MAPPINGS=None)
-    def test_is_config_loaded_with_no_config(self):
-        self.assertFalse(is_code_owner_mappings_configured(), "Mappings should not be configured.")
```

Comment:
**Reviewer Note:** FYI: I removed these tests of an internal method. Equivalent cases are already tested on other public calls that us this method.
Please mark as resolved once you've read this.